docs/release-notes/winter-2019/README.md

```diff
@@ -35,6 +35,10 @@ If you are not already on `@rei/cedar` >= 2.x.x, you will first need to [upgrade
 
 We have resolved an issue with CdrRating where it treated the `count` property differently depending on whether it was passed as a string or a number. Now CdrRating will show the review text any time it receives a value for `count`, and will not show any review text if it does not receive a value. You should not need to make any changes to your application code to support this change.
 
+### Pagination events and vue-router support
+
+Pagination functionality has been simplified and only emits a single `navigate` event. Responsive navigation behavior for the select no longer has to be manually attached. Vue-router example with router-link via scoped slots has been added. Previous and next links are always present (in a "disabled" state when appropriate). Minor style updates for hover and currently selected page.
```

Comment:
might want to mention like "API for scoped slots now uses an `attrs` object for easier binding"